snforge_std/src/cheatcodes/execution_info/contract_address.cairo

```diff
@@ -0,0 +1,31 @@
+// These cheatcodes are currently used only internally by the `interact_with_state` cheatcode,
```

Comment:
Is it technically possible to use them at all? If so, I'd adjust the naming so it's more "scary".